azavea/modellab-geoprocessing
# Make NDVI, VARI, VGI Colors Opaque

The NDVI, VARI, and VGI colors have been made opaque.  This corrects their appearance.
### Testing Instructions
- Build the jar `./sbt assembly`
- Copy the jar to the worker VM `scp ./target/scala-2.10/modellab-geoprocessing-assembly-0.4.1.jar vagrant@33.33.42.82:`
- Over-write the current geoprocessing 0.4.1 jar with this one
- Reload the worker
- Use the site and confirm that the NDVI, VARI, and VGI tiles look better

![screenshot from 2015-11-30 14 22 21](https://cloud.githubusercontent.com/assets/11281373/11482051/9d4aa616-976e-11e5-88f8-e1e8fa19182d.png)
Connects #38
